### DISCUSSION TITLE
Show frame range

### DISCUSSION THREAD

**Discussion by toddwdraper** [mifi/lossless-cut#1669] [Ideas]
I would find it very useful to easily see the range of frames in each segment. (e.g. Duration xxx, xxx ms, xxx frames (2283-5723)

**Comment by mifi** [comment]
does this setting help?

<img width="682" alt="Screenshot 2023-07-27 at 10 11 43" src="https://github.com/mifi/lossless-cut/assets/402547/9769d992-80c5-49a0-9fd5-158ac69ff2d6">
